modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -51,15 +71,57 @@ See link:https://nmstate.io/examples.html#interfaces-ethernet[NMState] for addit
                 next-hop-interface: <NIC1_name>
 ----
 <1> Add NMState YAML syntax to configure host interfaces.
-
-[TIP]
-====
-Consider saving the `networkConfig` YAML syntax to a file and testing it using the NMState command line interface before including it in the `install-config.yaml` file, because the installer will not check the NMState YAML syntax. Execute `nmstatectl gc <yaml-config>` to test the syntax. Errors in the YAML syntax might result in a failure to apply the network configuration. Additionally, maintaining the validated YAML syntax is useful when applying changes using Kubernetes NMState after deployment or when expanding the cluster.
-====
-
-The most common use case for this functionality is to specify a static IP address on the `baremetal` network, but you can also configure other networks such as a storage network. This functionality will also support other NMState features such as VLAN, VXLAN, bridges, bonds, routes, MTU, and DNS resolver settings.
-
++
 [IMPORTANT]
 ====
 Once deployed, you cannot modify the `networkConfig` configuration setting of `install-config.yaml` file to make changes to the host network interface. Use the Kubernetes NMState Operator to make changes to the host network interface after deployment.
 ====
++
+In the case of deploying {product-title} without a DHCP server on bare-metal network, use the following procedure to configure a static IP address for the bootstrap VM using Ignition:
```

Comment:
This is no longer necessary in the later 4.11 builds. 
https://bugzilla.redhat.com/show_bug.cgi?id=2048600